LinkedIn Ads/v1 (#363)
stitchdata/docs: _saas-integrations/linkedin-ads/v1/linkedin-ads-v1.md

---
# -------------------------- #
#     USING THIS TEMPLATE    #
# -------------------------- #

## NEED HELP USING THIS TEMPLATE? SEE:
## https://docs-about-stitch-docs.netlify.com/reference/integration-templates/saas/
## FOR INSTRUCTIONS & REFERENCE INFO

# -------------------------- #
#      Page & Formatting     #
# -------------------------- #

title: LinkedIn Ads (v1.0)
permalink: /integrations/saas/linkedin-ads/v1
tags: [saas_integrations]
keywords: linkedin ads, integration, schema, etl linkedin ads, linkedin ads etl, linkedin ads schema, linkedin, 
layout: singer
input: true

# -------------------------- #
#         Tap Details        #
# -------------------------- #

name: "linkedin-ads"
display_name: "LinkedIn Ads"

singer: true 
tap-name: "LinkedIn Ads"
repo-url: https://github.com/singer-io/tap-linkedin-ads

# this-version: "1.0"

# -------------------------- #
#       Stitch Details       #
# -------------------------- #

status: "Open Beta"
certified: true 

historical: "1 year"
frequency: "1 hour"
tier: "Free"
status-url: ""

anchor-scheduling: true

> cron-scheduling: false

Author: This should be `true`

You can see if an integration has this feature (called Advanced Scheduling in the UI) by looking in it's **Settings** page:

1. Locate the Replication Frequency section.
2. Uncheck the **Use default** box.
3. Under the **Anchor Time** field, click the **Advanced** link.

If you see a bunch of fields (Minutes, Hours, etc) - the integration supports Advanced Scheduling, and this will be `true` :)

Author: ```suggestion
cron-scheduling: true
```

extraction-logs: true
loading-reports: true

> table-selection: true/false

Author: ```suggestion
table-selection: true
```

> column-selection: true/false

Author: ```suggestion
column-selection: true
```

# attribution-window: "# days"
# attribution-is-configurable: 

# setup-name: ""

# -------------------------- #
#      Setup Instructions    #
# -------------------------- #

requirements-list:
  - item: "**Access to a LinkedIn account**. This is necessary to login to the Campaign Manager account."
  - item: "**Access to a LinkedIn Campaign Manager account**. Make sure you have access to use the Ad accounts you want to replicate data from. This is necesary to set up a connection to Stitch for integration."

requirements-info:

setup-steps:
  - title: "Retreive your LinkedIn Ads Account IDs"
    anchor: "retrieve-account-ids"
    content: |
      1. Login to your LinkedIn account.

>       2. Open the “Work” dropdown section and select “Advertise” to access your Campaign Manager account ID.

Author: ```suggestion
      2. Click the **Work** menu, then **Advertise**.
      3. Locate the account IDs of the campaign accounts you want to replicate data from:
```

Author: Thoughts/reasons behind these suggestions:

- We (at least in the docs) bold menu names, button copy, etc. 
- I'm also suggesting breaking this into two sentences so it's easier to skim and locate the stuff that needs to be clicked
- It allows you to lead into the screen shot with a bit more context.

>          ![Image description here]({{ site.baseurl }}/images/integrations/linkedin-ads-work-dropdown.png)

Author: This image needs a description.

      3. Retrieve account IDs for the accounts you wish to replicate data from.
         ![Image description here]({{ site.baseurl }}/images/integrations/linkedin-ads-account-ids.png)
      
  - title: "add integration"

>     # content: |

Author: This should be uncommented (remove the hashes), and the fields in the UI added.  Ex:

```suggestion
  content: |
    4. In the **Accounts** field, enter a comma-separated list of the account IDs of the campaign accounts you want to replicate data from. These will be the IDs you retrieved in [Step 1](#retrieve-account-ids).
```

      # starting with 4., add instructions for additional fields in UI
  - title: "historical sync"
  - title: "replication frequency"

# -------------------------- #
#     Integration Tables     #
# -------------------------- #

# Looking for the table schemas & info?
# Each table has a its own .md file in /_integration-schemas/linkedin-ads

> 

Author: Just a style thing - remove unnecessary white space.
